docs/swagger.yaml

```diff
@@ -120,11 +149,59 @@ components:
         instance:
           type: string
           description: UUID for the request
+    MerchantRedirectQrRequest:
+      title: MerchantRedirectQrRequest
+      type: object
+      properties:
+        id:
+          type: string
+          nullable: true
+        targetUrl:
+          type: string
+          format: uri
+          nullable: true
+      required:
+        - id
+        - targetUrl
+    MerchantRedirectQrResponse:
+      title: MerchantRedirectQrResponse
+      type: object
+      properties:
+        id:
+          type: string
+        qrUrl:
```

Comment:
Should we call this qrUrl? I think we should keep the body similar to the other endpoint, where we only call it "url"
What do you think @Sigvedash @annemsa75 ?

Reply:
QrUrl er jo url som genererer Qr, synes det er greit som det er, men kan overtales til kun url

Reply:
in that case, I think we should use `url` in both the response and the request

Reply:
MAybe we should rename the url in the request body to redirectUrl to keep more in line with what that parameter is doing. 
I agree that the link to the image in the response ideally shouldnt be called "url" but the cat is out the bag now. I think its better to keep the naming we have than make any differences between the two endpoints

Reply:
On the OTP QR endpoint, the request uses only url in the payload as well 🤔  So maybe we should just update both to be consistent?